frontend/src/container/Sidebar.jsx

```diff
@@ -90,6 +91,19 @@ export class Sidebar extends React.Component {
     )
   }
 
+  createLink = (label, to) => {
```

Comment:
Just a suggestion of a more explicit name, if you prefer `createLink` you can resolve this conversation
```suggestion
  getSidebarItem = (label, to) => {
```

Reply:
Changed.